src/blocks/HelloWorld/helloWorld.native.js

```diff
@@ -1,9 +1,14 @@
 // @flow
-import { Text } from "react-native";
+import { Text, View } from "react-native";
+import React from "react";
 import { helloStyle } from "./";
+import { Auth } from "../Biometrics";
```

Comment:
Auth is ambiguous and would lead to confusion as get to accounts. Lets make this a longer / more clear component name